scripts/release

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+PUBLIC_BUCKET="trussworks-tools-us-west-2"
+GITHUB_USER="trussworks"
+GITHUB_REPO="terraform-aws-iam-sleuth"
+ZIP_NAME="deployment.zip"
+NEW_TAG="1.0.7"
+
+ghr -u $GITHUB_USER -r $GITHUB_REPO $NEW_TAG $ZIP_NAME
```

Comment:
should probs do a check to see ift his is installed on someone's machine (if they run it locally/blindly)